tests/runtime/http_server/test_http_server.py

```diff
@@ -466,6 +466,36 @@ def test_inference_streaming_sample_module(sample_task_model_id, runtime_http_se
         )
 
 
+def test_inference_streaming_sample_module_throws(
+    sample_task_model_id, runtime_http_server
+):
+    """Check for testing an error unary-stream case"""
+    with TestClient(runtime_http_server.app) as client:
+        json_input = {
+            "model_id": sample_task_model_id,
+            "inputs": {"name": "world"},
+            "parameters": {"err_stream": True},
+        }
+        stream = client.post(
+            f"/api/v1/task/server-streaming-sample",
+            json=json_input,
+        )
+        assert stream.status_code == 200
```

Comment:
is it expected that the status code remains 200 even if the stream errors out? 🤔

Reply:
I think this would be expected for streams